opsdroid/connector/slack/__init__.py

```diff
@@ -309,3 +311,32 @@ def __init__(self, config, opsdroid=None):
                 )
 
         return aiohttp.web.Response(text=json.dumps("Received"), status=200)
+
+    @register_event(opsdroid.events.NewRoom)
+    async def _send_room_creation(self, creation_event):
+        params = creation_event.room_params
+        params = params.get("slack", params)
```

Comment:
you aren't doing anything with these params, and didn't we drop this anyway?

Reply:
Yep, think that's a bad merge :+1: